src/transformers/generation/utils.py

```diff
@@ -351,6 +359,43 @@ def prepare_inputs_for_generation(self, *args, **kwargs):
             "A model class needs to define a `prepare_inputs_for_generation` method in order to use `.generate()`."
         )
 
+    def set_cache_implementation(self, cache_implementation: Union[CacheImplementation, str], **kwargs):
+        """
+        Simple API to switch cache implementation in a model. Users could also do
+        `model.config.generation_config.cache_implementation = xxx` but they will most likely unexpected
+        behaviour
+
+        Args:
+            cache_implementation (`Union[CacheImplementation, str]`):
+                The target cache implementation
+            kwargs (`dict`, *optional*):
+                Optional key word arguments to be passed. E.g. for SinkCache, it is required to
```

Comment:
```suggestion
                Optional key word arguments to be passed. E.g. for "sink", it is required to
```
This might be clearer as we only deal with e.g. "static", "sink" and "dynamic" in this function